informer/integration/test/deletionerror/config_map.go

```diff
@@ -0,0 +1,58 @@
+// +build k8srequired
+
+package deletionerror
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+func createConfigMap(ID string) error {
```

Comment:
The e2e test is basically copied from the existing one and just modified. That made the setup pretty easy. I do not mind the refactoring of the common setup code now. We can do that later. I want to focus on the PM issue at hand.